phpstan.neon

```diff
@@ -30,8 +30,12 @@ parameters:
 
     ignoreErrors:
         - '#Class "Rector\\Nette\\Rector\\Class_\\MoveFinalGetUserToCheckRequirementsClassMethodRector" is missing @see annotation with test case class reference#'
+        - '#Class "Rector\\Nette\\Rector\\Class_\\CreateOrUpdateFormDataRector" is missing @see annotation with test case class reference#'
```

Comment:
The `CreateOrUpdateFormDataRectorTest` is needed

Reply:
Test added, you can check again